environment/src/test/java/com/sequenceiq/environment/environment/validation/validators/EncryptionKeyUrlValidatorTest.java

```diff
@@ -0,0 +1,29 @@
+package com.sequenceiq.environment.environment.validation.validators;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import com.sequenceiq.cloudbreak.validation.ValidationResult;
+
+class EncryptionKeyUrlValidatorTest {
+
+    private final EncryptionKeyUrlValidator underTest = new EncryptionKeyUrlValidator();
```

Comment:
It is a best practice to have a new instance created for both the test victim and all the mocks. Since this class does not need mocking, I suggest removing the `final` modifier and initializing `underTest` in a `@BeforeEach void setUp()` method.